### ISSUE TITLE
Gallery stuck in "Please wait"

### ISSUE THREAD

**Issue by HakanL** [smeighan/nutcracker#58]
I tried to click any of the buttons but nothing is ever displayed, just Please Wait.

**Comment by kgustafson** [comment]
There is a known issue with Ajax (the javascript code behind gallery) that cannot distinguish between www.localhost.com and localhost.com (or nutcracker123.com and www.nutcracker123.com).  Lose the www. before the call and it should function properly.  However, this is still a bug in Ajax and something that i need to address.

Kurt

**Comment by HakanL** [comment]
Yes, it worked without www. I believe some links to your site includes www, perhaps you can add a redirect on your side to help other users?
